frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -97,13 +93,21 @@ export class Wizard {
   }
 
   async selectWorkloadProfile(workloadProfile: string) {
-    await selectOptionByText(view.workloadProfileSelect, workloadProfile);
+    await selectItemFromDropdown(view.workloadProfileSelect, view.dropDownItem(workloadProfile));
   }
 
   async selectProvisionSource(provisionOptions) {
-    await selectOptionByText(view.provisionSourceSelect, provisionOptions.method);
+    await selectItemFromDropdown(
+      view.provisionSourceSelect,
+      view.bootSourceDropDownItem(provisionOptions.method),
+    );
     if (Object.prototype.hasOwnProperty.call(provisionOptions, 'source')) {
-      await fillInput(view.provisionSourceInputs[provisionOptions.method], provisionOptions.source);
+      if (provisionOptions.method === ProvisionSourceItem.URL) {
```

Comment:
is this necessary? can we keep the original form?

Reply:
like this: 
```
await selectItemFromDropdown(view.diskSource, view.diskDropDownItem(disk.source || DISK_SOURCE.Blank))
```

Reply:
ok. will do

Reply:
The reason is the value of `provisionOptions.method` for `URL` is `URL (adds disk)`, provisionSourceInputs only has `URL` and `Container`, so do it this way.